membership.go

```diff
@@ -197,10 +197,7 @@ func (m *membership) sudoGetTokenFromAccountID(w http.ResponseWriter, r *http.Re
 		return
 	}
 
-	id := ""
-
-	_, r.URL.Path = ShiftPath(r.URL.Path)
-	id, r.URL.Path = ShiftPath(r.URL.Path)
+	id := getURLPart(r.URL.Path, 3)
```

Comment:
the index for this should be 2 if I'm not mistaken, the URL for this is as follow: `/sudogettoken/:accountId`

Reply:
Sorry for long answer i'll test it now

Reply:
Yep it was really mistake